docs/src/mdx-components.tsx

```diff
@@ -82,7 +91,14 @@ export const th = ({ children, align }: React.ThHTMLAttributes<HTMLTableHeaderCe
   </TableCell>
 );
 export const ul = ({ children }: React.HTMLAttributes<HTMLUListElement>) => (
-  <ul style={{ listStyle: "inside none disc" }}>{children}</ul>
+  <ul
```

Comment:
can you also add spacing between components? it looks kinda sticky :)

Reply:
Added some